### PR TITLE
update vllm-docker-quick-start for vllm0.6.2

### PR DESCRIPTION
**Pull request by ACupofAir**
## Description

change example code for starting vllm serving, add `--distributed-executor-backend ray` parameter

### CODE REVIEW

File: docs/mddocs/DockerGuides/vllm_docker_quickstart.md [hunk 1]
```diff
@@ -240,17 +240,20 @@ Modify the `--load-in-low-bit` value to `fp6`, `fp8`, `fp8_e4m3` or `fp16`
...
+  --gpu-memory-utilization 0.9 \
+  --device xpu \ 
   --dtype float16 \
   --enforce-eager \
   --load-in-low-bit sym_int4 \
-  --max-model-len 4096 \
-  --max-num-batched-tokens 10240 \
+  --max-model-len 2048 \
+  --max-num-batched-tokens 4000 \
   --max-num-seqs 12 \
```
Comment (gc-fu) [inline docs/mddocs/DockerGuides/vllm_docker_quickstart.md:253]: We can probably also get rid of this max-num-seqs

File: docs/mddocs/DockerGuides/vllm_docker_quickstart.md [hunk 2]
```diff
@@ -742,16 +746,38 @@ vLLM Serving can be deployed as a server that implements the OpenAI API protocol
...
-  ...
+  --trust-remote-code \
+  --gpu-memory-utilization 0.9 \
+  --device xpu \
+  --dtype float16 \
+  --enforce-eager \
+  --load-in-low-bit fp8 \
+  --max-model-len 2048 \
+  --max-num-batched-tokens 4000 \
+  --max-num-seqs 12 \
```
Comment (gc-fu) [inline docs/mddocs/DockerGuides/vllm_docker_quickstart.md:777]: Same here

**Review by liu-shaojun — APPROVED** [review]
LGTM

**Review by gc-fu — APPROVED** [review]
LGTM

**Pull request merged**